src/main/java/com/quizapp/enterprise/controllers/GameController.java

```diff
@@ -28,7 +28,7 @@
 
 
     @PostMapping(value = "/newGame/{quizId}")
-    public Game startNewGame(@PathVariable("quizId") int quizId){
+    public Game startNewGame(@PathVariable("quizId") int quizId) throws Exception {
```

Comment:
related to respective handling of null pointer exception change

Reply:
Two thoughts:
-  Is an exception actually thrown here?   If not, this is  unused code, which is clutter, which is technical debt.
- Exception is an unchecked exception, which gets thrown automatically.  Thus, throws Exception on this signature isn't adding any value over what would occur by default.